sonar/modules/documents/jsonschemas/documents/document-v1.0.0_src.json

```diff
@@ -700,6 +696,7 @@
               "bf:Doi",
               "bf:Ean",
               "bf:Gtin14Number",
+              "bf:Iark",
```

Comment:
I did not find `bf:Iark` in the bibframe vocabulary. I would change this to simply `ARK` or use the existing type `uri`.

Reply:
:+1: